kanidm_tools/Cargo.toml

```diff
@@ -43,6 +43,8 @@ time = { version = "0.2", features = ["serde", "std"] }
 qrcode = { version = "0.12", default-features = false }
 bundy = "0.1"
 
+reqwest = { version = "0.11" }
```

Comment:
Why is this needed?

Reply:
It's not, was used temporarily while trying to unwrap errors... removing it. :)